### PR TITLE
Add sequential auction setup guide for Protected Audience

### PR DESCRIPTION
**Pull request by kevinkiklee**
This PR adds the sequential auction setup guide for Protected Audience

Preview link: https://pr-7809-static-dot-dcc-staging.uc.r.appspot.com/docs/privacy-sandbox/protected-audience-api/sequential-auction-setup/

### CODE REVIEW

File: site/en/docs/privacy-sandbox/protected-audience-api/sequential-auction-setup/index.md [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+  Learn how the Protected Audience auction can fit into the existing ad infrastructure
+description: >
+  Learn how the Protected Audience auction can fit into the existing ad infrastructure
+date: 2023-11-16
+authors:
+  - kevinkiklee
+  - garimamimani
+---
+
+Publishers typically diversify their ad demand sources to optimize for revenue and invoke multiple companies (for example, publisher ad servers, supply side platforms, and demand side platforms) to determine the best ad for a given ad slot on the page. Header bidding allows publishers to capture bids for an ad slot from a variety of demand sources. In a sequential auction setup, header bidding library may be used to run an auction with contextual data and Protected Audience is used to run an auction with cross-site data.
```
Comment (nancymic2) [inline site/en/docs/privacy-sandbox/protected-audience-api/sequential-auction-setup/index.md:14]:
```suggestion
Publishers typically diversify their ad demand sources to optimize for revenue and invoke multiple companies (for example, publisher ad servers, supply-side platforms, and demand-side platforms) to determine the best ad for a given ad slot on the page. Header bidding allows publishers to capture bids for an ad slot from a variety of demand sources. In a sequential auction setup, header bidding library may be used to run an auction with contextual data and Protected Audience is used to run an auction with cross-site data. 
```

File: site/en/docs/privacy-sandbox/protected-audience-api/sequential-auction-setup/index.md [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+   </td>
+   <td>A service used by publishers to manage and choose ads to be rendered on the site. A Publisher Ad Server may combine its own auction results, header bidder responses, direct-sold inventory, and more, to determine the ad that will provide the most revenue to a publisher. 
+<p>
+A Publisher Ad Server may provide a client-side library for interacting with the server.
+   </td>
+  </tr>
+  <tr>
+   <td>Top-level seller
+   </td>
+   <td>The party that invokes (i.e., creates) the Protected Audience multi-seller auction and participates in the top-level auction.
```
Comment (nancymic2) [inline site/en/docs/privacy-sandbox/protected-audience-api/sequential-auction-setup/index.md:105]:
```suggestion
   <td>The party that invokes (that is, creates) the Protected Audience multi-seller auction and participates in the top-level auction. 
```

File: site/en/docs/privacy-sandbox/protected-audience-api/sequential-auction-setup/index.md [hunk 3]
```diff
@@ -0,0 +1,255 @@
...
+   <td>Component seller
+   </td>
+   <td>The party that runs a component auction within the Protected Audience multi-seller auction to sell the publisher’s ad space to the buyers. Commonly a supply-side platform (SSP). 
+   </td>
+  </tr>
+</table>
+
+## Sequential auction setup
+
+In a sequential auction setup, the contextual auctions are executed first, then the Protected Audience auction is executed. This setup allows publishers to maximize their earning potential by running an auction with the contextual data available on the page, and also running an auction with cross-site data in a secure environment to protect user’s privacy.
```
Comment (nancymic2) [inline site/en/docs/privacy-sandbox/protected-audience-api/sequential-auction-setup/index.md:118]:
```suggestion
In a sequential auction setup, the contextual auctions are executed first, then the Protected Audience auction is executed. This setup allows publishers to maximize their earning potential by running an auction with the contextual data available on the page, and also running an auction with cross-site data in a secure environment to protect users' privacy. 
```

File: site/en/docs/privacy-sandbox/protected-audience-api/sequential-auction-setup/index.md [hunk 4]
```diff
@@ -0,0 +1,255 @@
...
+
+A [header bidding](https://docs.prebid.org/overview/intro-to-header-bidding.html#introduction-to-header-bidding) library may be executed first on the page to collect bids for the Publisher Ad Server’s contextual auction. Then, the adjusted winning bid price of the contextual auction can be entered into the Protected Audience auction as a bid floor. During the scoring step, the top-level seller can drop component auction bid prices below the bid floor by assigning them a zero score when the desirability score is calculated. If no Protected Audience component auction bid is above the bid floor, then the contextual auction winning ad is rendered to the user. If the Protected Audience auction returns a winner, it means it is above the bid floor, and the Protected Audience winning ad is rendered to the user. 
+
+{% Aside %}
+The setup in this document describes one potential approach that Protected Audience can be integrated with existing ad serving mechanisms, but it is not the only way. This document describes how one of the header bidding libraries, Prebid.js, has integrated with Protected Audience.
```
Comment (nancymic2) [inline site/en/docs/privacy-sandbox/protected-audience-api/sequential-auction-setup/index.md:123]:
```suggestion
The setup in this document describes one potential approach to integrating Protected Audience with existing ad serving mechanisms, but it is not the only way. This document describes how one of the header bidding libraries, Prebid.js, has integrated with Protected Audience.
```

File: site/en/docs/privacy-sandbox/protected-audience-api/sequential-auction-setup/index.md [hunk 5]
```diff
@@ -0,0 +1,255 @@
...
+  </figcaption>
+</figure>
+
+In a sequential auction setup, all contextual auctions are executed before the Protected Audience auction runs. In the setup explained in this document, we run a header bidding contextual auction by Prebid.js that feeds into a server-side auction by the Publisher Ad Server. 
+
+The publisher first initiates a header bidding contextual auction by calling Prebid.js with [a flag](https://docs.prebid.org/dev-docs/modules/fledgeForGpt.html#:~:text=bidderRequest.fledgeEnabled) to note that a Protected Audience will be executed afterwards. Then Prebid.js [collects](https://docs.prebid.org/dev-docs/modules/fledgeForGpt.html) the bid responses and sends them to the Publisher Ad Server for a server-side contextual auction. During the bid response collection step, the buyers and sellers have the opportunity to provide component auction configs and buyers’ signals (`perBuyerSignals`) to be used for the subsequent Protected Audience auction, if they wish to participate. That component auction config will eventually pass into the subsequent Protected Audience auction.
```
Comment (nancymic2) [inline site/en/docs/privacy-sandbox/protected-audience-api/sequential-auction-setup/index.md:175]: note that a Protected Audience will be executed afterwards...  protected audience what? missing word?

Comment (kevinkiklee) [reply site/en/docs/privacy-sandbox/protected-audience-api/sequential-auction-setup/index.md]: good catch!  updated.

File: site/en/docs/privacy-sandbox/protected-audience-api/sequential-auction-setup/index.md [hunk 6]
```diff
@@ -0,0 +1,255 @@
...
+
+<figure>
+  {% Img src="image/hVf1flv5Jdag8OQKYqOcJgWUvtz1/tMwANg0vcHve1toQ8ggq.png", alt="The Publisher Ad Server client-side library renders the ad that was chosen between the contextual auction and the Protected Audience auction", width="800", height="488" %}
+  <figcaption>
+    Ad rendering sequence on the publisher page<br/>
+    <a href="https://wd.imgix.net/image/hVf1flv5Jdag8OQKYqOcJgWUvtz1/tMwANg0vcHve1toQ8ggq.png" target="_blank">View full size diagram</a>
+  </figcaption>
+</figure>
+
+After the component auctions from the previous section are executed, the browser runs top-level seller’s scoring logic on each component auction’s winning ad. In the top-level seller’s `scoreAd()` function, the contextual auction adjusted bid price may be available as `sellerSignals`, and the scoring logic may compare that contextual auction bid price against the Protected Audience component auction’s winning bid price.
```
Comment (nancymic2) [inline site/en/docs/privacy-sandbox/protected-audience-api/sequential-auction-setup/index.md:235]:
```suggestion
After the component auctions from the previous section are executed, the browser runs the top-level seller’s scoring logic on each component auction’s winning ad. In the top-level seller’s `scoreAd()` function, the contextual auction adjusted bid price may be available as `sellerSignals`, and the scoring logic may compare that contextual auction bid price against the Protected Audience component auction’s winning bid price. 
```

File: site/en/docs/privacy-sandbox/protected-audience-api/sequential-auction-setup/index.md [hunk 7]
```diff
@@ -0,0 +1,255 @@
...
+  <figcaption>
+    Ad rendering sequence on the publisher page<br/>
+    <a href="https://wd.imgix.net/image/hVf1flv5Jdag8OQKYqOcJgWUvtz1/tMwANg0vcHve1toQ8ggq.png" target="_blank">View full size diagram</a>
+  </figcaption>
+</figure>
+
+After the component auctions from the previous section are executed, the browser runs top-level seller’s scoring logic on each component auction’s winning ad. In the top-level seller’s `scoreAd()` function, the contextual auction adjusted bid price may be available as `sellerSignals`, and the scoring logic may compare that contextual auction bid price against the Protected Audience component auction’s winning bid price. 
+
+If the contextual auction’s winning bid price is higher than the component auction’s winning bid price, then the `scoreAd()` function can return a desirability score of `0`. If there are no ads with desirability score higher than `0`, then it signifies that the contextual auction’s winning ad is more valuable than any of the component auction’s winning ads, and the `runAdAuction()` function returns `null`.
```
Comment (nancymic2) [inline site/en/docs/privacy-sandbox/protected-audience-api/sequential-auction-setup/index.md:237]:
```suggestion
If the contextual auction’s winning bid price is higher than the component auction’s winning bid price, then the `scoreAd()` function can return a desirability score of `0`. If there are no ads with a desirability score higher than `0`, then it signifies that the contextual auction’s winning ad is more valuable than any of the component auction’s winning ads, and the `runAdAuction()` function returns `null`. 
```

**Review by nancymic2 — APPROVED** [review]
LGTM with those little nits

**Review by tunetheweb — APPROVED** [review]

**Pull request merged**